packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -343,6 +334,18 @@ export function TableNG(props: TableNGProps) {
   );
 }
 
+function rowHeight({ row, columns, rows }) {
```

Comment:
I need help here. I know this is the ONLY right place to set row height. But I can't get access to real column width here and anywhere.
cc @adela-almasan @drew08t let's mob on it.

Reply:
maybe https://github.com/adazzle/react-data-grid/blob/dd76e81273a289f8f22065c47df714c24756f078/src/hooks/useColumnWidths.ts#L8

Reply:
UPD: we decided to set a reference to the header cells to track cells width.